app-backend/app/src/main/scala/bucket/Routes.scala

```diff
@@ -95,6 +97,39 @@ trait BucketRoutes extends Authentication
                   }
                 }
               }
+            } ~
+            post {
+              (authenticate & entity(as[Seq[UUID]])) { (user, sceneIds) =>
+                if (sceneIds.length > BULK_OPERATION_MAX_LIMIT) {
+                  complete(StatusCodes.RequestEntityTooLarge)
+                }
+
+                onSuccess(Buckets.addScenesToBucket(sceneIds, bucketId)) { scenes =>
+                  complete(scenes)
+                }
+              }
+            } ~
+            put {
+              (authenticate & entity(as[Seq[UUID]])) { (user, sceneIds) =>
+                if (sceneIds.length > BULK_OPERATION_MAX_LIMIT) {
+                  complete(StatusCodes.RequestEntityTooLarge)
+                }
+
+                onSuccess(Buckets.replaceScenesInBucket(sceneIds, bucketId)) { scenes =>
```

Comment:
Based on discussion in [here](https://github.com/azavea/raster-foundry/pull/564) (which seems to be accepted ?) this is more directly written as: 

```scala
complete { Buckets.replaceScenesInBucket(sceneIds, bucketId) }
```